app/src/main/java/cm/aptoide/pt/view/MainActivity.java

```diff
@@ -85,6 +106,14 @@
     return getIntent();
   }
 
+  @Override public void setUpdatesNumber(Integer updates) {
+    updatesNumber.setText(String.valueOf(updates));
```

Comment:
shouldn't this method set the visibility to VISIBLE? If you have 0 updates and do a hide and the new update arrives you won't see the notification badge, even tho you set it to 1.